### PR TITLE
[test-integration] Verify operator secret and service removal

### PR DESCRIPTION
**Pull request by Crumby**
Adds `*-generated-admin-secret` and `*-admin-service` removal verification upon **Infinispan** deletion

### CODE REVIEW

File: test-integration/operator-tests/src/main/java/org/infinispan/util/CleanUpValidator.java [hunk 1]
```diff
@@ -20,7 +20,9 @@ public CleanUpValidator(OpenShift openShift, String appName) {
       conditions = new ArrayList<>();
       conditions.add(
             () -> openShift.apps().statefulSets().withName(appName).get() == null &&
+                  openShift.secrets().withName("example-infinispan-generated-admin-secret").get() == null &&
```
Comment (dmvolod) [inline test-integration/operator-tests/src/main/java/org/infinispan/util/CleanUpValidator.java:23]:
```suggestion
                  openShift.secrets().withName(appName + "-generated-admin-secret").get() == null &&
```

Comment (Crumby) [reply test-integration/operator-tests/src/main/java/org/infinispan/util/CleanUpValidator.java]: Nice catch, thanks.

**Review by dmvolod — APPROVED** [review]

File: test-integration/operator-tests/src/main/java/org/infinispan/util/CleanUpValidator.java [hunk 2]
```diff
@@ -20,7 +20,9 @@ public CleanUpValidator(OpenShift openShift, String appName) {
       conditions = new ArrayList<>();
       conditions.add(
             () -> openShift.apps().statefulSets().withName(appName).get() == null &&
+                  openShift.secrets().withName(appName + "-generated-admin-secret").get() == null &&
```
Comment (dmvolod) [inline test-integration/operator-tests/src/main/java/org/infinispan/util/CleanUpValidator.java:23]: Secret name has been changed, @Crumby please rename it in the validation

Comment (Crumby) [reply test-integration/operator-tests/src/main/java/org/infinispan/util/CleanUpValidator.java]: Done

**Comment by dmvolod** [comment]
Thanks, @Crumby 
Merged into master

**Pull request merged**